docs/Load_Cell.md

```diff
@@ -0,0 +1,102 @@
+# Load Cells
```

Comment:
If a new document is added then it's also necessary to update docs/Overview.md and docs/_klipper3d/mkdocs.yml

Reply:
Will do